ports/espressif/supervisor/port.c

```diff
@@ -537,6 +540,17 @@ void port_post_boot_py(bool heap_valid) {
     }
 }
 
+
+#if CIRCUITPY_CONSOLE_UART
```

Comment:
Nice! So this redirects the IDF logging out of the console UART?

Reply:
right, the idf logging is preserved even after the CP console is up.